File: JS/NomeNoFormulario.js
/**
 * @version JAVASCRIPT
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 */

const YES = "YES";
const NO  = "NO";

let palavra = gets();

console.log( checkNameSize(palavra) );

/**
 * Check if the size of name is valid.
 */
function checkNameSize(name) {
    let result = NO;

    if ( name.trim().length < 81 ) result = YES;

    return result;
}
module.exports = {checkNameSize}

File: GO/NomeNoFormulario.go
/**
 * @version GO
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 */
package GO

import "strings"

const YES = "YES"
const NO = "NO"

/**
 * Check if the size of name is valid.
 */
func checkNameSize(name string) string {
	result := NO

	if len(strings.TrimSpace(name)) < 81 {
		result = YES
	}

	return result
}
